services/cd-service/pkg/sqlitestore/README.md

```diff
@@ -0,0 +1,3 @@
+# sqlitestore
```

Comment:
please add to readme_developers: what do I need to install to run kuberpult locally?
Installing "sqlite3" does not seem to be enough